dev/io.openliberty.checkpoint/src/io/openliberty/checkpoint/internal/CheckpointImpl.java

```diff
@@ -59,60 +68,102 @@
     @Activate
     public CheckpointImpl(ComponentContext cc) {
         this.cc = cc;
+        this.checkpointFeatures = "features".equals(cc.getBundleContext().getProperty(BootstrapConstants.CHECKPOINT_PROPERTY_NAME));
+        this.checkpointApplications = "applications".equals(cc.getBundleContext().getProperty(BootstrapConstants.CHECKPOINT_PROPERTY_NAME));
     }
 
     // only for unit tests
     CheckpointImpl(ComponentContext cc, ExecuteCRIU criu) {
         this.cc = cc;
         this.criu = criu;
+        this.checkpointFeatures = "features".equals(cc.getBundleContext().getProperty(BootstrapConstants.CHECKPOINT_PROPERTY_NAME));
```

Comment:
These calls are not necessary for unit tests to set the checkpointFeatures and checkpointApplications variable.  I would think the unit tests would fail with these calls here.